iet.so/src/iet_worker.cpp

```diff
@@ -114,73 +114,66 @@ IETWorker::IETWorker():endtest(false) {
 IETWorker::~IETWorker() {
 }
 
-
-
-/**
- * @brief logs the Gflops computed over the last log_interval period
- * @param gflops_interval the Gflops that the GPU achieved
- */
-void IETWorker::log_interval_gflops(double gflops_interval) {
-    string msg;
-    msg = " GPU flops :" + std::to_string(gflops_interval);
-    rvs::lp::Log(msg, rvs::logtrace);
-    log_to_json(IET_LOG_GFLOPS_INTERVAL_KEY, std::to_string(gflops_interval),
-                rvs::loginfo);
-
-}
-
-void IETWorker::blasThread(int gpuIdx,  uint64_t matrix_size, std::string  iet_ops_type, 
-    bool start, uint64_t run_duration_ms, int transa, int transb, float alpha, float beta,
-    int iet_lda_offset, int iet_ldb_offset, int iet_ldc_offset, int iet_ldd_offset) {
+void IETWorker::computeThread(void) {
 
     std::chrono::time_point<std::chrono::system_clock> iet_start_time, iet_end_time;
-    double timetakenforoneiteration;
-    double gflops_interval;
-    double duration;
-    uint64_t gem_ops;
+    double duration = 0;
     std::unique_ptr<rvs_blas> gpu_blas;
-    rvs_blas *free_gpublas;
-    string msg;
+    uint64_t _matrix_size_a;
```

Comment:
inconsistent naming convention for local vars, see if you can have without starting underscores